### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by dubstard**
```
"0vvvvuniswap.top",
"0vvwvuniswap.top",
"0vvwwuniswap.top",
"0vwwuniswap.top",
"daapscontractlink.live",
"kucoin-sne.online",
"manualcoonectsysnode.com",
"metablack.app",
"metasafe.live",
"sysnconnectinfo.manualcoonectsysnode.com",
"sysnconnectinfo.online",
"waconsyncdap.wadapconsyn.com",
"wadapconsyn.com",
```
image
image
image
image
image
image
image
Red Line Stealer RAT masquerading as "METAMASK BLACK" 😂

How it started                                      How It's going
image

It arrives via discord spam, [the fraudulent domain has already been revoked](https://twitter.com/dubstard/status/1484745169790554116)
image

### CODE REVIEW

**Comment by dubstard** [comment]
image
Not sure what is wrong, JSON parser complains

**Review by dubstard — COMMENTED** [review]
tried to see what is wrong with line 
    "sysnconnectinfo.online",
it looks good to me, not sure why the parser complains

**Comment by jacogr** [comment]
On it...

File: all.json [hunk 1]
```diff
@@ -2490,6 +2499,8 @@
     "synmultiwallet.com",
     "syntropynet-rewards-programs-claims-erc20-token.com",
     "synwalletbridge.com",
+    "sysnconnectinfo.manualcoonectsysnode.com"
```
Comment (jacogr) [inline all.json:2502]:
```suggestion
    "sysnconnectinfo.manualcoonectsysnode.com",
```

**Comment by jacogr** [comment]
Just as a side-note - when adding, you don't need to sort them. It will re-sort anyway when building, so after CI is done after a merge it is in sorted form.

(Obviously depends on your workflow... for me it is simpler just adding one to the top, but I also don't deal with your crazy volumes...)

**Review by jacogr — APPROVED** [review]

File: all.json (hunk 1 above)
Comment (dubstard) [reply all.json]: uff, sorry :(

**Comment by dubstard** [comment]
Thanks!

**Pull request merged**